Makefile

```diff
@@ -149,6 +168,20 @@ tools: $(CONTROLLER_GEN) \
 lint: $(GOLANGCI_LINT)
 	$(GOLANGCI_LINT) run ./... --fix
 
+.PHONY: jsonnet-fmt
+jsonnet-fmt: $(JSONNETFMT_BIN)
```

Comment:
there is no target - `$(JSONNNETFMT_BIN)

Reply:
good catch I forgot after I renamed the variable